.github/workflows/ci_pipeline.yml

```diff
@@ -110,47 +97,41 @@ jobs:
           - Second_orderSDC
           - soft_failure
           - TOMS
-        python: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        # python: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        python: ['3.10']
         exclude:
           - env: parallelSDC
             python: '3.11'
           - env: parallelSDC
             python: '3.12'
-
```

Comment:
I should have asked this in the last PR. But pytest allows for conditional skips. I believe we could add to specific tests to be skipped for specific python versions. This would clean up the pipeline a little bit. But it has its own disadvantages. What do you think? Since I am on my phone I cannot look up if this really works.

Reply:
We can also move the skipping of some jobs to the job-level instead of the  workflow level. I didn't know about this option. The downside is that it is not directly obvious that some tests are skipped, but (as an upside) more code is tested with more python-version.